c/tools/Makefile

```diff
@@ -0,0 +1,19 @@
+all: lug
+
+C_FLAGS += -Wall
+# C_FLAGS += -Wextra
```

Comment:
Commented-out code shouldn't be checked in

Reply:
Resolved with same commit as above, c2b1fc443ab3751c45ddf3e67b46bcf06f24d464.